frontend/src/firebase/firebaseInteractor.ts

```diff
@@ -365,13 +365,23 @@ export default class FirebaseInteractor {
       await this.updateCurrentUser({
         sessionId: sessionId < 8 ? ((sessionId + 1) as SessionId) : sessionId,
         onAssessment: false,
-        currentInterventionOrAssessment: documents.docs.filter(
-          (doc) => doc.data().session === sessionId + 1
-        )[0]?.id,
+        currentInterventionOrAssessment:
+          documents.docs.filter(
+            (doc) => doc.data().session === sessionId + 1
+          )[0]?.id || "",
```

Comment:
Why is the `|| ""` added?

Reply:
the third point - if it's undefined when u finish the last quiz of the assessment, if it's undefined, firebase doesn't support undefined data type

Reply:
If im not missing anything, if you don't have that and do the last quiz of the _LAST_ assessment, then it will have an update error in console